modules/oc-mirror-enclave-support.adoc

```diff
@@ -0,0 +1,127 @@
+// Module included in the following assemblies:
+//
+// * installing/disconnected_install/installing-mirroring-disconnected-v2.adoc
+
+:_mod-docs-content-type: Procedure
+[id="oc-mirror-enclave-support_{context}"]
+== How to mirror to an enclave
+
+When you mirror to an enclave, you must first transfer the necessary images from one or more enclaves into the enterprise central registry.
+
+The central registry is situated within a secure network, specifically a disconnected environment, and is not directly linked to the public internet. But the user must execute `oc mirror` in an environment with access to the public internet.
+
+.Procedure
+
+. Before running oc-mirror plugin v2 in the disconnected environment, create a `registries.conf` file.
++
+It is recommended to store the file under `$HOME/.config/containers/registries.conf` or `/etc/containers/registries.conf`.
```

Comment:
Should this be a note? Using the word "recommended" can be tricky and best to avoid. 

![Screenshot from 2024-06-20 16-45-47](https://github.com/openshift/openshift-docs/assets/57954076/02134fc0-5faa-4fac-b028-14a3c559381e)